src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Services/Interfaces/IEnterpriseUserAuthenticationService.cs

```diff
@@ -0,0 +1,20 @@
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+using Altinn.Platform.Authentication.Model;
+
+namespace Altinn.Platform.Authentication.Services.Interfaces
+{
+    /// <summary>
+    /// Interface handling methods for operations related to authentication of enterpriseusers
+    /// </summary>
+    public interface IEnterpriseUserAuthenticationService
+    {
+        /// <summary>
+        /// Method that fetches a response from authentication on sblbridge
+        /// </summary>
+        /// <param name="credentials">Credentials for an enterpriseuser</param>
+        /// <param name="bridgeApiEndpoint">SBLbridge-endpoint</param>
+        /// <returns>User profile connected to given ssn.</returns>
+        ConfiguredTaskAwaitable<HttpResponseMessage> GetResponseMessage(EnterpriseUserCredentials credentials, string bridgeApiEndpoint);
```

Comment:
`bridgeApiEndpoint` should be determined by the service itself, instead of having callers figure it out. See the constructor for `OrganisationsService` on how to inject HttpClient, logger and options.

Reply:
Thank you, re-written this code in newest commit.